lib/SlackBot.js

```diff
@@ -345,7 +345,7 @@ function Slackbot(configuration) {
              * we do NOT want to respond to incoming webhooks or slash commands
              * as the response can be used by developers to actually deliver a reply
              */
-            if (!message.command && !message.trigger_word && !message.submission) {
+            if (!message.command && !message.trigger_word && !message.submission && message.type !== 'view_submission') {
```

Comment:
this means devs need to `bot.res.send('')` themselves because we don't have a `modal.Ok()` or `modal.error()` function.